DQMOffline/JetMET/interface/JetAnalyzer_HeavyIons.h

```diff
@@ -91,10 +91,20 @@ class JetAnalyzer_HeavyIons : public DQMEDAnalyzer {
   void bookHistograms(DQMStore::IBooker &, edm::Run const &, edm::EventSetup const &) override;
   
  private:
-  
+  static const Int_t fourierOrder_ = 5;
```

Comment:
hi @kurtejung - please use int rather than Int_t. Second, constexpr instead of static const would be more appropriate.